AccommodationService/Contracts/Enums/PricingOption.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace AccommodationService.Contracts.Enums;
+
+public enum PricingOption

```

Comment:
It's better for this to be part of domain, but it is not big of an issue leaving it here